setup.py

```diff
@@ -5,7 +5,7 @@
                         open('dateparser/__init__.py').read()).group(1)
 
 introduction = re.sub(r':members:.+|..\sautomodule::.+|:class:|:func:|:ref:',
-                      '', open('docs/introduction.rst').read())
+                      '', open('docs/introduction.rst', encoding='utf-8').read())
 history = re.sub(r':mod:|:class:|:func:', '', open('HISTORY.rst').read())
```

Comment:
I believe you will also need to add utf-8 encoding for `HISTORY.rst` as well.

Reply:
Yes! you are correct. Thanks :smile: